hyperspy/_signals/eds_tem.py

```diff
@@ -242,3 +243,88 @@ def get_calibration_from(self, ref, nb_pix=1):
         if mp_ref.has_item("Detector.EDS.live_time"):
             mp.Acquisition_instrument.TEM.Detector.EDS.live_time = \
                 mp_ref.Detector.EDS.live_time / nb_pix
+
+    def vacuum_mask(self, threshold=1.0, closing=True):
+        """
+        Generate mask of the vacuum region
+
+        Parameters
+        ----------
+        threshold: float
+            For a given pixel, maximum value in the energy axis below which the
+            pixel is considered as vacuum.
+        closing: bool
+            If true, applied a morphologic closing to the mask
+
+        Return
+        ------
+        mask: signal
+            The mask of the region
+        """
+        from scipy.ndimage.morphology import binary_dilation, binary_erosion
+        mask = (self.max(-1) <= threshold)
+        if closing:
+            mask.data = binary_dilation(mask.data, border_value=0)
+            mask.data = binary_erosion(mask.data, border_value=1)
```

Comment:
[same comment]:
> This closing only gets rid of False holes in the mask (i.e. a single False pixel, surrounded by True on most  sides). Is it intended to leave the True holes? Two more reversed identical operations would get rid of them (i.e. first dilation, then erosion, so that the whole treatment would look like dilation -> erosion -> erosion -> dilation)